src/Redux/Books/boooks-redux.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'bookstore/src/redux/books/addBook';
```

Comment:
- **[OPTIONAL]** I think this file name should be just `books.js` since its parent folder `redux` gives it enough context already.